profiler/src/Demos/NoOpWcfService/Properties/launchSettings.json

```diff
@@ -1,23 +1,30 @@
 {
   "profiles": {
-    "Demos.NoOpWcfService": {
+    "NoOpWcfService-Alone": {
+      "commandName": "Executable",
+      "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\NoOpWcfService\\$(TargetFramework)\\Datadog.Demos.NoOpWcfService.exe",
+      "nativeDebugging": true
+    },
+    "NoOpWcfService+Profiler": {
       "commandName": "Executable",
       "executablePath": "$(BaseOutputPath)\\$(ConfigBasedRelativeOutputPath)\\profiler\\src\\Demos\\NoOpWcfService\\$(TargetFramework)\\Datadog.Demos.NoOpWcfService.exe",
       "environmentVariables": {
         "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{BD1A650D-AC5D-4896-B64F-D6FA25D6B26A}",
```

Comment:
Obviously up to you, but IMO you'd be better off using the native loader in these demos etc too

Reply:
I did not update these WCF samples used to reproduce a customer issue a long time ago